### DISCUSSION TITLE
How to add a YTM playlist?

### DISCUSSION THREAD

**Discussion by chicocanada** [z-huang/music#1106] [Q&A]
Hello,

I can't seem to be able to add a YTM playlist, but only YT.

Thanks!

**Comment by RenautMestdagh** [comment]
Add playlist to your YTM account.
In Innertube, go to home->account. Press a playlist and press the box icon with an arrow in it

**Reply by chicocanada** [reply]
I'm sorry, I should have specified.

When adding a list by URL I can only add a YT list, not a YTM list.
